src/Raven.Server/Documents/Handlers/Admin/Processors/Indexes/AbstractAdminIndexHandlerProcessorForPut.cs

```diff
@@ -77,6 +80,17 @@ public override async ValueTask ExecuteAsync()
                         $"Index name must not start with '{Constants.Documents.Indexing.SideBySideIndexNamePrefix}'. Provided index name: '{indexDefinition.Name}'");
                 }
 
+                if (indexDefinition.SourceType != IndexSourceType.Documents)
```

Comment:
is this the only flow? some validation is also done by IndexCreateProcessor

Reply:
moved to `AbstractIndexCreateController.ValidateStaticIndexAsync`